tools/cli/admin_cluster_commands_test.go

```diff
@@ -134,3 +137,348 @@ func TestValidSearchAttributeKey(t *testing.T) {
 	assert.Error(t, visibility.ValidateSearchAttributeKey("9lives"))
 	assert.Error(t, visibility.ValidateSearchAttributeKey("tax%"))
 }
+
+func TestAdminDescribeCluster(t *testing.T) {
+	// Initialize mock controller
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	// Create mock frontend and admin clients
+	serverFrontendClient := frontend.NewMockClient(mockCtrl)
+	serverAdminClient := admin.NewMockClient(mockCtrl)
```

Comment:
let's avoid defining mocks at top level like this. If one test case's expectation fails it will fail the rest. better to define these inside inner t.Run for isolation purposes